src/content/docs/en/basics/layouts.mdx

```diff
@@ -14,6 +14,8 @@ We conventionally use the term "layout" for Astro components that provide common
 
  But, there is nothing special about a layout component! They can [accept props](/en/basics/astro-components/#component-props) and [import and use other components](/en/basics/astro-components/#component-structure) like any other Astro component. They can include [UI frameworks components](/en/guides/framework-components/) and [client-side scripts](/en/guides/client-side-scripts/). They do not even have to provide a full page shell, and can instead be used as partial UI templates.
 
+If a layout component contains a page shell, the `<html>` element must be the parent of all other elements in the component. All [`<style>`](https://docs.astro.build/en/guides/styling/#styling-in-astro) or [`<script>`](https://docs.astro.build/en/guides/client-side-scripts/#using-script-in-astro) elements must be enclosed by the `<html>` tags.
```

Comment:
```suggestion
However, if a layout component does contain a page shell, its `<html>` element must be the parent of all other elements in the component. All [`<style>`](/en/guides/styling/#styling-in-astro) or [`<script>`](/en/guides/client-side-scripts/#using-script-in-astro) elements must be enclosed by the `<html>` tags.
```
Just tried to connect it to the paragraph above more, and used the link format we need here in Docs.